part-one/Optimism.md

```diff
@@ -8,31 +8,29 @@ The fundamental tenor of this book is one of optimism. This is in part a reflect
-
-There is an extreme position that would suggest we would have been better off never developing knowledge. That we would still live in a state of paradise had we not tasted the forbidden fruit. Not only is it hard to see how we would go back there now, but more importantly, I for one prefer not to be consumed by wild animals.
+There is an extreme position that would suggest we would have been better off never developing knowledge {{ "FOX" | cite}}. That we would still live in a state of paradise had we not tasted the forbidden fruit. Not only is it hard to see how we would go back there now, but more importantly, I for one prefer not to be consumed by wild animals.
 
 Will all future problems be solvable, including say climate change? There is, of course, no guarantee. We might wind up with a problem we cannot solve and that might cause our extinction. But what is certain is that assuming that problems cannot be solved guarantees that they will not be solved. Pessimism is a self-defeating attitude when it leads to inaction.
 
-Yes, digital technologies including the internet and advances in automation have brought with them a new set of problems. We will encounter many in this book, including immense pressure on people&apos;s ability to earn a living and the conflicts arising from being exposed to content that runs counter to one&apos;s upbringing or deeply held cultural or religious beliefs.
+Yes, digital technologies including the Internet and advances in automation have brought with them a new set of problems. We will encounter many in this book, including immense pressure on people&apos;s ability to earn a living and the conflicts arising from being exposed to content that runs counter to one&apos;s upbringing or deeply held cultural or religious beliefs.
 
 And yet this newly inflated space of what is possible also includes amazing progress, such as the example of free diagnoses of diseases for all of humanity encountered at the end of the previous chapter.
 
-Believing in the potential for real progress though is not the same as being a Pollyanna. Progress does not happen by itself as a deterministic function of technology. Contrary to Kevin Kelly&apos;s claims in his book &ldquo;What Technology Wants&rdquo;, technology doesn&apos;t want anything by itself and certainly not a better world for humanity. It simply makes such a world possible. 
+Believing in the potential for real progress though is not the same as being a Pollyanna. Progress does not happen by itself as a deterministic function of technology. Contrary to Wired magazine co-founder Kevin Kelly&apos;s claims in his book &ldquo;What Technology Wants&rdquo;, technology doesn&apos;t want anything by itself and certainly not a better world for humanity. It simply makes such a world possible. 
```

Comment:
We can footnote Kevin Kelly if we want to but he's done a lot more than co-found Wired